### ISSUE TITLE
Markdown interprets <?php as comment

### ISSUE THREAD

**Issue by PascalNegwer** [BoostIO/Boostnote#275]
Hi,
i tried to use the Markdown Codeblocks for xml and php highlighting, but in preview the <?php and <?xml blocks are interpreted as comments.

## Code:
<img width="295" alt="bildschirmfoto 2017-02-08 um 12 00 30" src="https://cloud.githubusercontent.com/assets/15419498/22734724/031fc784-edf7-11e6-9de9-3a55ee04ddd3.png">

## Expected Result
```php
<?php
/**
 * asdfg
 * asdf
 */
private function asdf() {
  echo 'Hallo Welt';
}
echo 'Hallo Welt';
?>
```

## Result
<img width="309" alt="bildschirmfoto 2017-02-08 um 12 00 41" src="https://cloud.githubusercontent.com/assets/15419498/22734751/19d4b6e2-edf7-11e6-9ce6-9213e208f7ec.png">

**Comment by asmsuechan** [comment]
Hi @PascalNegwer, thanks for your report. We confirmed this issue on https://github.com/BoostIO/Boostnote/issues/205. We try to fix as soon as possible.

**Comment by kazup01** [comment]
Hi @PascalNegwer san, @asmsuechan fixed this bug on #341 🎉 We will add it at next update! Thank you for waiting!

**Comment by kazup01** [comment]
Sorry we can't release this feature at current version because it is too less time.
We'll release it next time. Thank you for your waiting! @PascalNegwer
